docs/sources/clients/promtail/stages/logfmt.md

```diff
@@ -0,0 +1,44 @@
+---
+title: logfmt
+---
+# `logfmt` stage
+
+The `logfmt` stage is a parsing stage that reads the log line as logfmt.
+
+## Schema
+
+```yaml
+logfmt:
+  # Maps set of keys/values extracted from the data
+  #
+  mapping:
+    [ <string>: <string> ... ]
+```
+
+This stage uses the go-kit/log structured logging parser to extract key/value pairs of data from log lines into the extracted map.
```

Comment:
nit: instead of referencing `go-kit/log` would be more appropriate to refer the original logfmt blog?
https://brandur.org/logfmt

Reply:
thanks, but I felt it more appropriate to refer to the implementation used, in case of any future bugs this would be a better reference imo

Reply:
You are right tho, I added an additional link to the brandur.org site